board/fischertechnik/TXT/rootfs/etc/sudoers.d/touchscreen-calibration

```diff
@@ -2,4 +2,4 @@
 ## for changing display settings
 
 Defaults!/usr/bin/ts_calibrate env_keep=TSLIB_TSDEVICE
-ftc     ALL = NOPASSWD: /usr/bin/ts_calibrate, /etc/init.d/S90launcher restart
+ftc     ALL = NOPASSWD: /sbin/calibrate-touchscreen
```

Comment:
Now you dropped too much - we still need to be able to restart the launcher in order to change the screen orientation...

```suggestion
ftc     ALL = NOPASSWD: /sbin/calibrate-touchscreen, /etc/init.d/S90launcher restart
```